src/Appwrite/Extend/Exception.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace Appwrite\Extend;
+
+use Utopia\Config\Config;
+
+class Exception extends \Exception
+{
+    /**
+     * Error Codes
+     */
+    const TYPE_NONE = '';
+
+    /** API */
+    const TYPE_PROJECT_NOT_FOUND       = 'project_not_found';
```

Comment:
Again, codes are very unclear. Let's have a pattern we can follow. It can be per status category (4xx, 5xx) and service (users, functions) or per entity instead of service to be able to reuse them across controllers.

Reply:
I have made an attempt at this. Please check and let me know